incubator/mariadb-cluster/values.yaml

```diff
@@ -61,6 +67,14 @@ master:
     ## Persistent Volume size
     ##
     size: 8Gi
+    ## Configure MySQL with a custom my.cnf file
+    ## ref: https://mysql.com/kb/en/mysql/configuring-mysql-with-mycnf/#example-of-configuration-file
+    ##
+    # config: |-
```

Comment:
Since we are allowing users  to completely override the `my.ini` configuration, I recommend adding the default configuration here, allowing users to fine tune the configuration.

Reply:
the indentation level of the `config` field needs to be fixed  here. Current indentation suggests that its a parameter if of `persistence`